src/Content/Text/Plaintext.php

```diff
@@ -116,7 +116,7 @@ public static function getBoundariesPosition($text, $open, $close, $occurrences
 	 * @see   \Friendica\Content\Text\BBCode::getAttachedData
 	 *
 	 */
-	public static function getPost($item, $limit = 0, $includedlinks = false, $htmlmode = BBCode::API, $target_network = '')
+	public static function getPost($item, $limit = 0, $includedlinks = false, $htmlmode = BBCode::MASTODON_API, $target_network = '')
```

Comment:
Can you here please try to add type-hints for all parameter?

Reply:
This is something either you or I can do later, let's move on with this PR.

Reply:
My point was, when he had already edited the line, he could have easily added the type-hints there (they are quickly added as they are not complicated like in other methods).

Reply:
You'll find out soon enough nothing is easy with Michael. 😛

He can be a pretty obstinate work horse, so this kind of consideration is better left to us to pick up in its wake.

Reply:
When changing any type hint, I always will check all associated functions, if there could be any problem doing so. This costs a lot of time. And especially in this case the main task to make the tests work again, so that we can safely merge PRs again.